dist/backdraft.js

```diff
@@ -1214,42 +1257,67 @@ _.extend(Plugin.factory, {
     // @col: The column from the column manager corresponding to the column we're creating
     //   filtering wrappers for.
     _createFilteringWrappers: function(head, col) {
+      window.activeMenu = null;
       var filter = col.filter;
+      var filterActive = filter.value || filter.eq || filter.gt || filter.lt;
       // create filtering wrapper div
       var wrapperDiv = document.createElement('div');
-      wrapperDiv.className = "DataTables_filter_wrapper";
+      wrapperDiv.className = "dropdown DataTables_filter_wrapper";
       wrapperDiv.id = "wrapper-" + col.attr;
-      wrapperDiv.innerHTML = "Filter";
+      wrapperDiv.innerHTML = this.filterButtonTemplate({ active: filterActive });
 
       // determine how many columns we need if we're dealing with list filtering
       var listClass = "";
       if (filter.type === "list") {
-        if (filter.options.length > 30)
-          listClass = " triple"
-        else if (filter.options.length > 15)
-          listClass = " double"
-        else
+        if (filter.options.length > 30) {
+          listClass = " triple";
+        } else if (filter.options.length > 15) {
+          listClass = " double";
+        } else {
           listClass = " single";
+        }
       }
 
       // create filtering menu div
       var filterDiv = document.createElement("div");
-      filterDiv.className = "filterMenu" + listClass;
+      filterDiv.className = "filterMenu dropdown-menu" + listClass;
       filterDiv.id = "menu-" + col.attr;
-      wrapperDiv.appendChild(filterDiv);
 
       // put filtering controls in filter div and put wrapper div in header
       if (filter.type === "string")
         $('input', head).appendTo(filterDiv)
       else
         $('ul', head).appendTo(filterDiv);
+      $('button', head).appendTo(filterDiv);
+      wrapperDiv.appendChild(filterDiv);
       head.appendChild(wrapperDiv);
 
-      // handle hovering on wrapperDiv
-      $('.DataTables_filter_wrapper', head).hover(function () {
-        $('.filterMenu', head).slideDown(200);
-      }, function () {
-        $('.filterMenu', head).slideUp(50);
+      // handle filter menu display
```

Comment:
I'd like to at least have you explore popover elements if you haven't already. By default they don't dismiss by clicking outside of them, or elements inside them (which I think is a good thing.. since you have an explicit cancel button)